library/src/lib/time/time.component.ts

```diff
@@ -38,6 +38,16 @@ export class TimeComponent implements OnChanges, ControlValueAccessor {
     /** @Input When set to false, hides the input for seconds. */
     @Input() displaySeconds: boolean = true;
 
+    /** @Input When set to false, hides the input for minutes. */
```

Comment:
can you remove all these `@Input`? It's leftover from the previous way docs were built